### PR TITLE
feat: initial-atago-branding

### PR DESCRIPTION
**Pull request by stevanlohja**

### CODE REVIEW

**Comment by drospa** [comment]
LGTM! Great job!

**Comment by drospa** [comment]
LGTM!

File: src/index.html [hunk 1]
```diff
@@ -2,25 +2,24 @@
...
-          Request will send 1 Test ETC to your designated {process.env.MANTIS_VM} address.<br />
+          Request will send 1 tFUEL to your designated receiving address.<br />
           Users can make 1 claim every 24 hours.
         </p>
         <form id="request-form">
-          <label for="addressField">Your {process.env.MANTIS_VM} address:</label>
+          <label for="addressField">Your receiving address:</label>
           <input type="text" id="address-field" name="address-field" />
-          <input type="submit" id="form-submit" value="Request test ETC" />
+          <input type="submit" id="form-submit" value="Request tFUEL" />
```
Comment (artur-biniek-iohk) [inline src/index.html:22]: Should it be **t**FUEL? I think "t" left from word "test"

**Review by artur-biniek-iohk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (stevanlohja) [reply src/index.html:22]: tFUEL is corret.

Comment (stevanlohja) [reply src/index.html:22]: I don't have repo permissions to resolve this thread.

**Pull request merged**